libobs/util/simde/README.libobs

```diff
@@ -1,5 +1,5 @@
-This is a slightly modified version of https://github.com/nemequ/simde/commit/cafec4b952fa5a31a51a10326f97c2e7c9067771
-sse{,2}.h and mmx.h was moved down from the original "x86" subdirectory,
-subsequently the '#include "../simde-common.h"' line in mmx.h was changed to '#include "simde-common.h"'
+This is a slightly modified version of the simde directory in
```

Comment:
Is the only modification now that unused files have been removed?

Reply:
Correct, now the files match the SIMDe repo, except for the reformatting by `clang-format`